spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/file-appender.xml

```diff
@@ -14,7 +14,7 @@ initialization performed by Boot
 		<file>${LOG_FILE}</file>
 		<rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
 			<cleanHistoryOnStart>${LOG_FILE_CLEAN_HISTORY_ON_START:-false}</cleanHistoryOnStart>
-			<fileNamePattern>${LOG_FILE}.%d{yyyy-MM-dd}.%i.gz</fileNamePattern>
+			<fileNamePattern>${LOG_FILENAME_PATTERN:-${LOG_FILE}.%d{yyyy-MM-dd}.%i.gz}</fileNamePattern>
```

Comment:
`LOG_FILENAME_PATTERN` is not a bad name but i wonder if it should somehow have the `ROLLING` notion in there since it's only used when rolling over log files according to the RollingPolicy ...  `ROLLING_FILENAME_PATTERN` perhaps?

@eddumelendez thanks for the quick response !

Reply:
We already have `FILE_LOG_PATTERN` that's mapped from `logging.pattern.file` so I agree that the system property needs to be more differentiated. How about `ROLLING_FILE_LOG_PATTERN`? We could then perhaps also introduce a new property named something like `logging.pattern.rolling-file` that maps to the new system property.

Reply:
Thanks guys I haven taken this comments into account.